src/components/templates/ArtPiece/ArtPiece.tsx

```diff
@@ -24,7 +21,7 @@ export interface ArtPieceProps {
 }
 
 export const ArtPiece: React.FC<ArtPieceProps> = ({ venue }) => {
-  const [showEventSchedule, setShowEventSchedule] = useState(false);
+  if (!venue) return <>Loading...</>;
```

Comment:
Can probably use the new `<Loading>` component.

Reply:
used `<Loading />` here: https://github.com/sparkletown/sparkle/pull/1426/commits/880f966fdbeb06065f5946456902e13410b88582